simphony_ui/tests/test_ui.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
Here you are now testing the local_traits mostly. I would rename the file, the class, and the associated docs.